src/components/Text/Text.stories.module.css

```diff
@@ -0,0 +1,40 @@
+/*------------------------------------*\
+    #TEXT STORIES
+\*------------------------------------*/
+
+/**
+ * 1) For the "color" story to display the different text color options.
+ * 2) Uses grid to neatly align the color variations.
+ */
+.color__table { /* 1 */
+  display: grid; /* 2 */
+  grid-template-columns: 1fr 1fr 1fr; 
+  gap: 4px;
```

Comment:
Maybe use `0.25rem` here instead of px? (I don't know if we ever decided on a pattern in EDS but in Along we only use px for things like borders and drop shadows; spacing and sizing is also rem. I don't think it has a user-facing effect though so it's more for code consistency imo.)

Reply:
Will change to `eds-size-half` (0.25rem), thanks